homeassistant/components/sleepiq/number.py

```diff
@@ -138,6 +162,18 @@ def _get_foot_warming_unique_id(bed: SleepIQBed, foot_warmer: SleepIQFootWarmer)
         get_name_fn=_get_foot_warming_name,
         get_unique_id_fn=_get_foot_warming_unique_id,
     ),
+    CORE_CLIMATE_TIMER: SleepIQNumberEntityDescription(
+        key=CORE_CLIMATE_TIMER,
+        native_min_value=30,
+        native_max_value=600,
```

Comment:
The max value can be taken from the SleepIQCoreClimate object when it's available below

Reply:
Since the max value is only defined on SleepIQFuzionCoreClimate, I can't use it unless I import the Fuzion class. Which would not make it backwards compatible. 

Would you want to make a change to the library to expose it on SleepIQCoreClimate?

Reply:
Yes, that should be added

Reply:
Submitted a PR.

Reply:
Thanks for the quick approval and release. Updated the library version, will make the change, test and have a new commit soon.

Reply:
Updated to use the max timer variable from SleepIQCoreClimate.